# Robot:
collision_box: [1.0, 1.0, 0.6]

# === Incremental Search ===
# factor_of_safeZone: 2.0  # replan_range = factor_of_safeZone * offset_dyn_obs
factor_of_safeZone: 3.5  # replan_range = factor_of_safeZone * offset_dyn_obs

# === Planner Control ===
REPLAN_STRATEG_Indicator: 2   # [0, 1, 2, 3] ...

# === Rewiring ===
Barrier_Resolution: 400
Barrier_Radius_Factor: 2.3
# Barrier_Radius_Factor: 1.15


# Dynamic Obstacle Parameter:
obs_radius: 0.2
obs_initial_x: -1.2    # Initial X Pos of Dyn Obs
obs_initial_y: 1.0     # Initial Y Pos of Dyn Obs
obs_movable_range: 2.0 # Dyn Obs Movement Range

# Refresh Rate
DynObs_rate: 0.2       # Dynamic Obstacle Publish Rate
# DynObs_rate: 0.5       # Dynamic Obstacle Publish Rate
PseudoRob_rate: 0.01   # Pseudo Robot Publish Rate
# PseudoRob_rate: 0.1    # Pseudo Robot Publish Rate

# Debugger Mode
Debug: True  # disables PID and enables pseudo robot
